transforms/angle-brackets/transform.test.js

```diff
@@ -68,6 +68,20 @@ test('boolean-values', () => {
   `);
 });
 
+test('string-preserve-quotes', () => {
+  let input = '{{foo-bar placeholder="Choose a \\"thing\\"..."}}';
+  // eslint-disable-next-line prettier/prettier
+  expect(runTest('string-preserve-quotes.hbs', input)).toMatch('<FooBar @placeholder="Choose a \\"thing\\"..." />');
+});
+
+test('string-mixed-line-endings', () => {
+  let input = '{{foo-bar placeholder="Choose a... \\t\\r\\n"}}';
+
+  expect(runTest('string-mixed-line-endings.hbs', input)).toMatch(
```

Comment:
@Turbo87 I had to change to `toMatch` from `toMatchInlineSnapshot` because `toMatchInlineSnapshot` runs prettier not allowing me to express the string correctly for the test.  I